cmd/main.go

```diff
@@ -257,6 +261,45 @@ func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
 
+	go func() {
+		kclient, err := mesherykube.New(nil)
```

Comment:
Pardon me, I am a bit rusty but are we assuming here that meshery is deployed within kubernetes?

Reply:
Thanks for pointing that out @utkarsh-pro . This implementation is not something that I intend to keep as is. There are also some other things apart from what you said such as ensuring that the operator is deployed at the startup (which also ensures the availability of MeshSync and broker) that we should consider. But, I haven't started working on those yet. So, I thought, let's introduce this change now, then put in a PR addressing these issues in a more organised manner.

Reply:
@utkarsh-pro , the constructor for the Kubernetes client (`mesherykube.New`) takes care of handling in-cluster vs out of cluster cases. It returns the client accordingly. For reference - meshkit [client.go](https://github.com/layer5io/meshkit/blob/42abdc0df310af02ec4f2bc380964a9fba4e4880/utils/kubernetes/client.go#L28)